settings.gradle

```diff
@@ -1,2 +1,2 @@
-include ':app', ':google-maps-services-java-master', ':google-maps-services-java-0.10.0'
+include ':app', ':google-maps-services-java-0.10.0'
```

Comment:
did you test it? it works fine without any issues?

Reply:
Ferdousara and I tried it and it seems to not affect the app